docs/dev/log_integration.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Create a Log Integration
+kind: documentation
+aliases:
+- /logs/faq/partner_log_integration
+further_reading:
+- link: "/integrations/#cat-log-collection"
+  tag: "Documentation"
+  text: "See existing Datadog Log integrations"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Learn about the Log Explorer"
+- link: "/logs/log_configuration/pipelines/"
+  tag: "Documentation"
+  text: "Learn about log pipelines"
+---
+
+## Overview
+
+This page walks Technology Partners through creating a Datadog Log integration.
+
+## Log integrations
+
+Use the [Logs Ingestion HTTP endpoint][1] to send logs to Datadog. 
+
+## Development process
+
+### Guidelines
+
+When creating a log integration, consider the following best practices:
```

Comment:
All of the points below are the suggested best practices, so not just the tags being lowercase. Maybe put this line before the start of the description list? And then have "The source and service tags must be in lowercase." as a single description?